### ISSUE TITLE
Add labels to standups, e.g. WIP

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#7094]
In standups, we often write `WIP` at the top of the card:

![standups-label](https://user-images.githubusercontent.com/39854876/185605419-810c8a20-16b2-4b13-a135-2755efd44e07.png)

I think it'd be more visually appealing if we could add labels. 

We could add a labels button to the left of the emojis button, the labels could look like GitHub's, and show up at the top right of the card.

### AC

- [ ] Discuss whether it's a bad idea
- [ ] Create a design issue if it's a good idea

**Comment by Dschoordsch** [comment]
For this particular case it seems a draft state would address the problem directly without adding too much complexity.

**Comment by nickoferrall** [comment]
True! A draft state would solve this tension and would be easier to implement. Although if we could filter updates by label, like GitHub, it could also be a helpful way of viewing Squad-specific updates, for example

**Comment by JimmyLv** [comment]
I had the same idea of designing a generic `Label/Tag` mechanism. That would allow filtering cards, which could be more than just this current meeting, but all of history.

**Comment by jmtaber129** [comment]
Closing this as the primary use case should be solved by https://github.com/ParabolInc/parabol/pull/7925!
